Info.plist

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>CFBundleExecutable</key>
+	<string>Cute3DSCapture</string>
+	<key>CFBundleIdentifier</key>
+	<string>com.gotos.cute3ds</string>
+	<key>CFBundleVersion</key>
+	<string>1.3</string>
```

Comment:
Can we import this string from another place? I think the makefile has the tool version hardcoded as well, which means we now have to places for it. It'd like to unify that, if possible.

Reply:
(It's in configure.ac, actually)

Reply:
Technically got a working solution using `VERSION=$(cat configure.ac | grep AC_INIT | awk '{print $2}' | tr -d ,)` and a shell script that generates the `Info.plist`, so now it only needs to be changed in one place.

I was trying to pass this variable down from the configure script, and have `make-macos-app.sh` invoked by make after linking, but couldn't figure out how to get it to work.

Reply:
I think I'm fine with this solution for now. Passing a variable sure would have been more elegant, but I'm not sure how to do that well either, so let's go with this. :)